composer.json

```diff
@@ -50,6 +50,7 @@
         "symfony/polyfill-php80": "^1.22",
         "symfony/polyfill-php81": "^1.26",
         "symfony/process": "^5",
+        "symfony/var-exporter": "^5.4",
```

Comment:
Why is got here?

Reply:
Yeah I noticed i'd added it to the dev packages where infact I actually use it in the recipe so need it as part of the main packages